### DISCUSSION TITLE
Question for team Algoritma (on Capstone DA project). I am pretty worried

### DISCUSSION THREAD

**Discussion by mynugraErlangga** [teamalgoritma/community#668] [Q&A] [answered]
Selamat malam

Saat saya menulis Q&A post ini (12.46 AM Sabtu 20 Mei), saya Baru rechecked repository saya untuk Capstone DA (Flask UI) di github ini:
![Screenshot 2023-05-20 004313](https://github.com/teamalgoritma/community/assets/125937571/e5021152-9fe8-4216-976d-a6b7d1bb55ca)

Issue: Setelah melihat "Initial commit" (Yang Telah saya lakukan 4 hari lalu), saya Tidak menemukan keseluruhan submitted files saya. Saya sangat yakin saya telah uploaded files-files tersebut berdasarkan video instruksi hingga menekan tombol hijau "Commit Changes":
![Screenshot 2023-05-20 004418](https://github.com/teamalgoritma/community/assets/125937571/11256cfe-dbc7-42a5-8e9c-fe17acdbccb1)

Dua scenarios:
A. Project files sedang dalam proses grading oleh Algoritma dan untuk sementara tidak terlihat di Github.

B. Walaupun saya yakin tidak benar   saya tidak menekan tombol hijau "Commit Changes" setelah uploaded keseluruhan project files saya. 

Pertimbangannya: Jika ternyata "B", mohon konsiderasi Algoritma untuk tidak deduct point saya Lebih jauh dari penalty wajar (memang keterlambatan 1 hari).
Alasan: Saya sudah meng-investasikan waktu dan usaha belajar untuk pengerjaan Capstone DA ini; Akan backfire/negate rasa self-accomplishment overall dan progress belajar saya, Jika di over-penalize karena overlook yang sangat kecil.

Terima kasih.

**Comment by nadyadtm** [comment]
Dear Pak @mynugraErlangga 
Baik pak, kami akan coba untuk konfirmasi terlebih dahulu

**Reply by nadyadtm** [reply, marked as answer]
Dear Pak @mynugraErlangga 
Untuk hal seperti ini tidak masalah. Penilaian tetap akan di -1 karena pengumpulan terlambat 1 hari. Bila ada yang ingin dikonfirmasikan bisa langsung membalas pesan ini. Terima kasih
